### PR TITLE
Update favicons and configurations to a simpler, more modern approach.

### PR DESCRIPTION
**Pull request by cookrn**
Ticket: https://app.dework.xyz/indiedao/lvl-protocol?taskId=955e7a2b-1d81-4beb-b92f-dd1b4b114b23

Article: https://evilmartians.com/chronicles/how-to-favicon-in-2021-six-files-that-fit-most-needs

This removes excess images/sizes that aren't needed anymore for a
majority of device support, including M$ customizations. Instead use the
newer web manifest standard and a few scalable sizes.

### CODE REVIEW

**Comment by cookrn** [comment]
Blocked by #91

File: pages/_document.js [hunk 1]
```diff
@@ -87,26 +87,17 @@ export default class MyDocument extends Document {
...
-              sizes={size}
-              key={size}
-            />
-          ))}
-          <meta name="msapplication-TileColor" content="#d7d7d7" />
-          <meta name="msapplication-TileImage" content="/browserconfig.xml" />
+          <link rel="icon" href="/images/favicons/favicon.ico" sizes="any" />
+          <link
+            rel="icon"
+            href="/images/favicons/icon.svg"
```
Comment (mckelvey) [inline pages/_document.js:93]: @cookrn — I’m getting a 404 on this in the Vercel preview... `.../favicons/favicon.svg`?

**Review by mckelvey — CHANGES_REQUESTED** [review]
Looks great @cookrn — just a little 404.

File: pages/_document.js (hunk 1 above)
Comment (cookrn) [reply pages/_document.js:93]: Good catch!

**Review by mckelvey — APPROVED** [review]
💯

**Pull request merged**